packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/QueueInfoColumn.tsx

```diff
@@ -162,17 +177,25 @@ const QueueInfoColumn: React.FC<QueueInfoColumnProps> = ({
       )}
 
       {/* mobile only */}
-      <div className="mt-3 flex items-center justify-between sm:hidden">
-        {!isStaff && hasDemos && buttons}
+      <div className="mt-5 flex items-center justify-around sm:hidden">
+        {!isStaff && buttons}
```

Comment:
oh yeah were you going to try having a large "Join Queue" button here? With width something like 80 or 90%?
![image](https://github.com/user-attachments/assets/9f68bb86-7e90-4ec1-970d-35458ac8b0bc)